tests/create-errors/enc/optee/user_ta_header_defines.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* Licensed under the MIT License. */
+#pragma once
```

Comment:
The OE coding standard requires the following style.
```
#ifndef _USER_TA_HEADER_DEFINES_H
#define _USER_TA_HEADER_DEFINES_H
...
#endif /* _USER_TA_HEADER_DEFINES_H */
```

Reply:
Missed it. -- File no longer exists.